src/systemd/sd-bus-vtable.h

```diff
@@ -187,6 +187,56 @@ struct sd_bus_vtable {
                 .x = { { 0 } },                                         \
         }
 
+#define _SD_ECHO(X) X
+#define _SD_CONCAT(X) X "\0"
```

Comment:
i think we should use a more specific prefix here, i.e. _SD_BUS_ or so... after all we might need something like this in other headers (i.e. outside of sd-bus) too one day...

Reply:
Do you mean for everything or just `_SD_ECHO` and `_SD_CONCAT`? All the iteration stuff isn't really tied to sd-bus in any way. If another library needs to loop over even varargs elements it can easily reuse the same code. When that happens, we can move this stuff to sd-common and share it between libraries (we could put it there immediately but it might make sense to wait with that until we actually need it in two places).